great_ape_safe/ape_api/badger.py

```diff
@@ -424,7 +424,8 @@ def set_gas_station_watchlist(self):
             if "executor" in label:
                 min_bal = 1e18
             if label == "upkeep_manager":
-                min_bal = 1e18
+                min_bal = 3e18
+                min_top_up = 2e18
```

Comment:
```suggestion
                min_bal = 5e18
                min_top_up = 4e18
```

Reply:
with gwei at 42 and topping up upkeep manager manually with 2 extra eth the script still reverts (ie mngr needs >2.6eth)

given that in the future average gwei might go up more, and that we might want to deploy other upkeeps at >2m gas, lets give upkeep manager a nice buffer.